docs/tex/getting-started.tex

```diff
@@ -34,9 +34,11 @@ \subsubsection{Your first Edward program}
 
 \begin{lstlisting}[language=Python]
 import numpy as np
+from edward.stats import norm
```

Comment:
we should use scipy's as a default; i think we'll deprecate `edward.stats` soon as it doesn't serve much of a purpose. e.g.,
```
from scipy.stats import norm
```